Tests/GoogleAITests/GenerateContentResponses/unary-success-unknown-enum-prompt-blocked.json

```diff
@@ -0,0 +1,23 @@
+{
+  "promptFeedback": {
+    "blockReason": "FAKE_NEW_BLOCK_REASON",
```

Comment:
I think this file name needs to be failure since it throws on a blocked prompt

Reply:
Done (also `finishReason` since it throws `GenerateContentError.responseStoppedEarly`).